x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -272,6 +272,7 @@ export const RISK_ENGINE_SETTINGS_URL = `${RISK_ENGINE_URL}/settings`;
 export const ASSET_CRITICALITY_URL = `/internal/asset_criticality`;
 export const ASSET_CRITICALITY_PRIVILEGES_URL = `/internal/asset_criticality/privileges`;
 export const ASSET_CRITICALITY_STATUS_URL = `${ASSET_CRITICALITY_URL}/status`;
+export const ASSET_CRITICALITY_CSV_UPLOAD_URL = `${ASSET_CRITICALITY_URL}/upload_csv`;
```

Comment:
I recommend to move `ASSET_CRITICALITY...` constants into Asset criticality domain. Create `common/asset_criticality` folder with `constants.ts`. It will help to define proper ownership avoid expanding `common/constants.ts` with not really common constants.